### ISSUE TITLE
Design and implement mechanism for when to show and when not to show banner

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-privacy-choice#23]
We also need to have a mechanism that shows the banner on the website or omits it.

The simplest case is if a user selects that GPC signals should be sent to all sites. In that case the banner is not shown anymore. 

Another case is if the user goes site-by-site. In that case we would need to check whether a site is in the domain list (and what its setting is). If the site is not in the domain list, the banner should be shown. If it is in the domain list, showing the banner depends on what the selection is in the domain list.

The bottom line is that there needs to be a check against the domain list. 

This is not fully thought out, but I wanted to flag this issue as it just occurred to me.

(assigning all three of you, @bella-tassone, @chunyuema, and @ekuller; we can decide later on who does it; or if any of you wants, feel free to pick up this issue)
